services/email/index.js

```diff
@@ -5,7 +5,7 @@ var nodemailer = require('nodemailer');
 var config = require('../../config');
 var i18nFuture = require('hof').i18n;
 var Hogan = require('hogan.js');
-var i18nLookup = require('hof').i18nLookup;
+var i18nLookup = require('i18n-lookup');
```

Comment:
i18n doesn't really do anything and will be removed when I have finished refactoring the email